src/arrayMethodReduce.js

```diff
@@ -5,7 +5,19 @@
  */
 function applyCustomReduce() {
   [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+    let accumulator = initialValue;
+    let startValue = 0;
```

Comment:
Rename this variable to startIndex, because this is the index of element of the array from which we will start in the loop

Reply:
Done 👌